dask/dataframe/io/tests/test_orc.py

```diff
@@ -75,8 +79,110 @@ def test_orc_single(orc_files):
 
 
 def test_orc_multiple(orc_files):
-    d = read_orc(orc_files[0])
-    d2 = read_orc(orc_files)
+    d = dd.read_orc(orc_files[0])
+    d2 = dd.read_orc(orc_files)
     assert_eq(d2[columns], dd.concat([d, d])[columns], check_index=False)
-    d2 = read_orc(os.path.dirname(orc_files[0]) + "/*.orc")
+    d2 = dd.read_orc(os.path.dirname(orc_files[0]) + "/*.orc")
     assert_eq(d2[columns], dd.concat([d, d])[columns], check_index=False)
+
+
+@pytest.mark.skipif(
+    parse_version(pa.__version__) < parse_version("4.0.0"),
+    reason=("PyArrow>=4.0.0 required for ORC write support."),
+)
```

Comment:
Okay - It looks like pyorc has some [libprotobuf-collision problems](https://github.com/noirello/pyorc/issues/10) that seem to crash the worker for python-3.9 on macOS.  For now, I am just skipping this test for python-3.9.  Is this reasonable @jrbourbeau ?

Reply:
oops - highlighted the wrong test